src/Microsoft.Azure.Functions.ExtensionBundle/NuGet.Config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <clear />
     <add key="azure-functions-extension-bundles" value="https://pkgs.dev.azure.com/azfunc/public/_packaging/azure-functions-extension-bundles-feed/nuget/v3/index.json" protocolVersion="3" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />

```

Comment:
Aren't we recommended to move away from directly using `nuget.org` and use this via a feed?